kong/api/routes/kong.lua

```diff
@@ -91,18 +91,23 @@ return {
         ngx.log(ngx.ERR, "could not get node id: ", err)
       end
 
+      local available_plugins = {}
+      for name in pairs(singletons.configuration.loaded_plugins) do
+        available_plugins[name] = require("kong.plugins." .. name .. ".handler").VERSION or true
```

Comment:
The plugin handlers are loaded into `_G.kong.db.plugins.handlers` at `init`, so as an alternative, you can also fetch the version this way:

```suggestion
        available_plugins[name] = kong.db.plugins.handlers[name].VERSION or true
```

To my eyes this seems a little cleaner than using `require` in this code path. Not really a strong opinion, so feel free to disregard!

Reply:
ha thx! was struggling to find that!